.github/workflows/allthekeeps-testnet.yml

```
@@ -0,0 +1,103 @@
+        with:
+          environment: 'ropsten'
+      
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "12.x"
+
+      - name: Run install repositories
+        run: ./install-repositories.sh
+      
+      - name: Get upstream packages versions
+        if: github.event_name == 'workflow_dispatch'
+        uses: keep-network/ci/actions/upstream-builds-query@v1
+        id: upstream-builds-query
+        with:
+          upstream-builds: ${{ github.event.inputs.upstream_builds }}
+          query: |
+            keep-core-contracts-version = github.com/keep-network/keep-core/solidity#version
+            keep-ecdsa-contracts-version = github.com/keep-network/keep-ecdsa/solidity#version
+            tbtc-contracts-version = github.com/keep-network/tbtc/solidity#version
+
+      - name: Deploy subgraph to thegraph.com
+        if: github.event_name != 'workflow_dispatch'
+        working-directory: ./keep-subgraph
+        run: |
+          CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY=${{secrets.CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY}} \
+          ETH_RPC_URL=${{secrets.ETH_RPC_URL}} \
+          SUBGRAPH_DEPLOY_KEY=${{secrets.SUBGRAPH_DEPLOY_KEY}} \
+          SUBGRAPH_SLUG=${{secrets.SUBGRAPH_SLUG}} \
+            ./deploy-subgraph.sh
+
+      - name: Deploy latest subgraph to thegraph.com
+        if: github.event_name == 'workflow_dispatch'
+        working-directory: ./keep-subgraph
+        run: | 
+          CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY=${{ secrets.CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY }} \
+          ETH_RPC_URL=${{ secrets.ETH_RPC_URL }} \ 
+          SUBGRAPH_DEPLOY_KEY=${{ secrets.SUBGRAPH_DEPLOY_KEY }} \ 
+          SUBGRAPH_SLUG=${{ secrets.SUBGRAPH_SLUG }} \ 
+          KEEP_CORE_VERSION=${{ steps.upstream-builds-query.outputs.keep-core-contracts-version }} \
+          KEEP_ECDSA_VERSION=${{ steps.upstream-builds-query.outputs.keep-ecdsa-contracts-version }} \
+          KEEP_TBTC_VERSION=${{ steps.upstream-builds-query.outputs.tbtc-contracts-version }} \
+            ./deploy-subgraph.sh
+
+      - name: YARN build
+        run: ./install-dapp.sh
+
+      - name: Deploy to GCP bucket
+        if: github.event_name == 'workflow_dispatch'
+        uses: thesis/gcp-storage-bucket-action@v3.1.0
+        with:
+          service-key: ${{ secrets.KEEP_TEST_CI_UPLOAD_DAPP_JSON_KEY_BASE64 }}
+          project: ${{ env.GOOGLE_PROJECT_ID }}
+          bucket-name: allthekeeps.test.keep.network
+          set-website: true
+          home-page-path: index.html
+          error-page-path: index.html
+          build-folder: build
+
+      - name: Notify CI about completion of the workflow
+        if: github.event_name == 'workflow_dispatch'
+        uses: keep-network/ci/actions/notify-workflow-completed@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.CI_GITHUB_TOKEN }}
+        with:
+          module: "https://github.com/keep-network/allthekeeps"
```

This should match ci configuration file https://github.com/keep-network/ci/pull/15/files#diff-ffb6b648a5b9ecb316c6b166af7cfc006edc09675cf2be200ad799180c92446aR51
```suggestion
          module: "github.com/keep-network/allthekeeps"
```

---

Done